app/static/css/style.css

```diff
@@ -14,10 +14,27 @@ body {
   margin: 0 auto;
 }
 
+.overlay {
+    display: none;
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0.8);
+    z-index: 2;
+}
+
 * {
   font-family: Lora;
 }
 
+*:focus {
```

Comment:
Looks like this accidentally got pulled in from #202. Could we rebase onto master to get rid of this?